pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -236,24 +237,8 @@ router.post('/', (req, res, next) => {
     }
 });
 
-function b64EncodeUnicode(str) {
-    // (1) use encodeURIComponent to get percent-encoded UTF-8
-    // (2) convert percent encodings to raw bytes
-    // (3) convert raw bytes to Base64
-    return Buffer.from(encodeURIComponent(str).replace(/%([0-9A-F]{2})/g, (match, p1) => {
-        return String.fromCharCode('0x' + p1);
-    })).toString('base64');
-}
-
-function b64DecodeUnicode(str) {
-    // Going backwards: from bytestream, to percent-encoding, to original string.
-    return decodeURIComponent(Buffer.from(str, 'base64').toString().split('').map((c) => {
-        return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
-    }).join(''));
-}
-
 function getHash(contents) {
-    return b64EncodeUnicode(sha256(contents).toString());
+    return b64Util.b64EncodeUnicode(sha256(contents).toString());
```

Comment:
Not really relevant to this PR, but why is the hash base-64 encoded?

Reply:
you're right, let's revisit this later ;-)